src/views/enroll-choices/FactorList.js

```diff
@@ -39,23 +39,33 @@ define([
     },
 
     children: function () {
-      if (this.model.get('enrolled')) {
-        return [['<span class="icon success-16-green"></span>', '.enroll-factor-label']];
+      var children = [];
+
+      if (this.options.optionalFactors) {
```

Comment:
Yea.. i can't really think of a good name for this variable.. It is not that the factor itself is optional, it could be required + enrolled, but still have an optional instance. I basically want the button to show for this case - https://github.com/okta/okta-signin-widget/pull/565/commits/a88df6511b9d74742aeaa7c0d6c9bdb7df746ae1#diff-7bd64fea6e4a7f4b128f4e3d2edcb935R117
Because that list is always for optional factors and should always show the setup button inline.. Maybe i should just rename it to `showSetupButton: true`

Reply:
I was thinking more on the lines on keeping something similar to what we have, so each row knows how to render itself based on the model values... but I just realize the problem with that, which is the case of a required factor that has all required enrolls but still has one optional... 

ok, then we can change the variable name to what you are suggesting... or maybe better is `showInlineSetupButton`